addons/api/addon/abilities/target.js

```diff
@@ -12,4 +12,18 @@ export default class TargetAbility extends ModelAbility {
   get canConnect() {
     return this.hasAuthorizedAction('authorize-session');
   }
+
+  /**
+   * @type {boolean}
+   */
+  get canRemoveHostSources() {
+    return this.hasAuthorizedAction('remove-host-sources');
+  }
+
+  /**
+   * @type {boolean}
+   */
+  get canRemoveCredentialSources() {
+    return this.hasAuthorizedAction('remove-credential-sources');
+  }
```

Comment:
Will `add-*-sources` permissions be added in the future? Apologies if they already exists and I missed them.

Reply:
Yes.  This PR is just for remove.